Documentation/devicetree/bindings/display/panel/panel-dsi.yaml

```diff
@@ -0,0 +1,114 @@
+# SPDX-License-Identifier: (GPL-2.0-only OR BSD-2-Clause)
+%YAML 1.2
+---
+$id: http://devicetree.org/schemas/display/panel/panel-dsi.yaml#
+$schema: http://devicetree.org/meta-schemas/core.yaml#
+
+title: Generic MIPI DSI Panel
+
+maintainers:
+  - Timon Skerutsch <timon@diodes-delight.com>
+
+allOf:
+  - $ref: panel-common.yaml#
+
+properties:
+  compatible:
+    description:
+      Shall contain a panel specific compatible and "panel-dsi"
+      in that order.
+    items:
+      - {}
+      - const: panel-dsi
+
+  dsi-color-format:
+    description: |
+      The color format used by the panel. Only DSI supported formats are allowed.
+    enum:
+      - RGB888
+      - RGB666
+      - RGB666_PACKED
+      - RGB565
+
+  lanes:
+    description: |
+      The number of lanes used by the panel.
+    $ref: /schemas/types.yaml#/definitions/uint32
+    enum: [1, 2, 3, 4]
+
+  mode:
+    description: |
+      DSI mode flags. See DSI Specs for details.
+      These are driver independent features of the DSI bus.
+    items:
+      - const: MODE_VIDEO
+      - const: MODE_VIDEO_BURST
+      - const: MODE_VIDEO_SYNC_PULSE
+      - const: MODE_VIDEO_AUTO_VERT
+      - const: MODE_VIDEO_HSE
+      - const: MODE_VIDEO_NO_HFP
+      - const: MODE_VIDEO_NO_HBP
+      - const: MODE_VIDEO_NO_HSA
+      - const: MODE_VSYNC_FLUSH
+      - const: MODE_NO_EOT_PACKET
+      - const: CLOCK_NON_CONTINUOUS
+      - const: MODE_LPM
+      - const: HS_PKT_END_ALIGNED
+
+  reg: true
+  backlight: true
+  enable-gpios: true
+  width-mm: true
+  height-mm: true
+  label: true
+  panel-timing: true
+  port: true
+  power-supply: true
+  reset-gpios: true
+  ddc-i2c-bus: true
+
+required:
+  - panel-timing
+  - reg
+  - power-supply
+  - dsi-color-format
+  - lanes
+
+additionalProperties: false
+
+examples:
+  - |
+    panel {
+        compatible = "panel-mfgr,generic-dsi-panel","panel-dsi";
+        label = "Simple DSI Panel";
```

Comment:
https://github.com/torvalds/linux/blob/master/Documentation/devicetree/bindings/display/panel/panel-common.yaml#L34-L46
>       If no such name exists, and unless the property is mandatory according to
>      device tree bindings, it shall rather be omitted than constructed of
>      non-descriptive information. For instance an LCD panel in a system that
>      contains a single panel shall not be labelled "LCD" if that name is not
>      inscribed on the system or used in a descriptive fashion in system
>      documentation.

Reply:
AFAIK it's never passed on to the user, and it's more for those analysing the DT to be able to understand which device is being derived.
Looking at the other bindings, very few define a label (amusingly panel/lgphilips,lb035q02.yaml includes `label = "lcd";` almost to spite the writer of the docs!), and a very quick grep in arch/arm/boot/dts doesn't appear to include any displays with labels.

Please drop it.

Reply:
thats fair, was mostly following the other panel-*.yaml bindings which make use of it but you are right its actually not passed to user space. I somehow remembered that it would be used for the display name in the window manager but that does not seem to be the case, at least not with Wayland. Does not have any value in that case.